### PR TITLE
Railgun Changes

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request
make railgun better at its job

## Why It's Good For The Game
currently you can just ignore railgun by standing in the middle of the laser, gives it minirocket stats for clearing weeds and area denial, increased cooldown to 5 minutes and spread to compensate

## Changelog
:cl:
code: railgun targeting changed from pick to rand
balance: increased railgun spread
balance: changed railgun to be explosive
balance: increased railgun cooldown to 5 minutes
balance: reduced time it takes to fire and land railgun
/:cl:

https://user-images.githubusercontent.com/98577688/166409471-847f21d9-d772-4210-b66a-ad3af1491310.mp4

### CODE REVIEW

**Comment by tyeagg** [comment]
[#10116](https://github.com/tgstation/TerraGov-Marine-Corps/pull/10116)

**Review by TiviPlus — CHANGES_REQUESTED** [review]
This should not be using CAS ammo period, why cant you just copy paste relevant code

File: code/__DEFINES/dropship_equipment.dm [hunk 1]
```diff
@@ -9,13 +9,14 @@
 ///This equipment can only be used in firemissions
 #define FIRE_MISSION_ONLY (1<<3)
 ///Whether we can interact with this equipment
-#define IS_INTERACTABLE (1<<4)
+#define IS_INTERACTABLE (1<<5)
```
Comment (TiviPlus) [inline code/__DEFINES/dropship_equipment.dm:12]: You need to undo this change idk why you did it

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Ill let you use rail gun ammo since you dont seem to be able to figure out how to make it non cas ammo

**Pull request merged**